client/src/components/Card.jsx

```diff
@@ -10,19 +10,36 @@ const Card = (props) => {
   */
   //console.log("😀",value);
   //console.log("😍",props);
+
+  const [background, setBackground] = useState('red');
+
+  useEffect(() => {
+   if (averageScore > 7) {
+    setBackground('green')
+   }
+   else if (averageScore > 5) {
+    setBackground('yellow')
+   }
+   else {
+    setBackground('red')
+   }
+  },[averageScore])
```

Comment:
nit: indentation appears to only be one space - the rest of the document uses two spaces